aten/src/ATen/native/native_functions.yaml

```diff
@@ -2289,6 +2289,8 @@
 
 - func: group_norm(Tensor input, int num_groups, Tensor? weight=None, Tensor? bias=None, float eps=1e-05, bool cudnn_enabled=True) -> Tensor
 
+- func: _group_norm_all_outputs(Tensor input, int num_groups, Tensor? weight=None, Tensor? bias=None, float eps=1e-05) -> (Tensor, Tensor, Tensor)
+
```

Comment:
Why do we need a new operator? Is it to save values so that we can compute the gradients of the input Tensor?

Reply:
Also, what is the difference between _group_norm_all_outputs and native_group_norm?

Reply:
Yes exactly to save the mean and variance so we can compute the gradients of the input tensor.

Difference between `_group_norm_all_outputs` and `native_group_norm` is that `_group_norm_all_outputs` will do [some validation](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/group_norm.cpp#L131-L152) (originally done in `group_norm`, now `group_norm` just calls `_group_norm_all_outputs`) before calling `native_group_norm`

Reply:
cc @albanD How do you feel about the functions added to native_funcitons? Any alternative ideas?